libs/designsystem/src/lib/components/modal/alert/alert.component.html

```diff
@@ -1,9 +1,9 @@
 <article #alertWrapper>
-  <h3 class="title">{{ title$ | async }}</h3>
   <kirby-empty-state
     [iconName]="iconName"
     [themeColor]="iconThemeColor"
     [subtitle]="message$ | async"
+    [title]="title$ | async"
```

Comment:
Consider readability:
```suggestion
    [title]="title$ | async"
    [subtitle]="message$ | async"
```